Actors/CounterActor/install.sh

```diff
@@ -20,4 +20,16 @@ cp dotnet-include.sh ./CounterActorApplicationCSharp/CounterActorWebServicePkg/C
 cp dotnet-include.sh ./CounterActorTestClient
 sfctl application upload --path $appPkg --show-progress
 sfctl application provision --application-type-build-path CounterActorApplicationCSharp
-sfctl application create --app-name fabric:/CounterActorApplicationCSharp --app-type CounterActorApplicationTypeCSharp --app-version 1.0.0
\ No newline at end of file
+if [ $# -eq 0 ]
+  then
+    echo "No arguments supplied, proceed with default instanceCount of 1"
+    sfctl application create --app-name fabric:/CounterActorApplicationCSharp --app-type CounterActorApplicationTypeCSharp --app-version 1.0.0 --parameters "{\"CounterActorWebService_InstanceCount\":\"1\"}"
```

Comment:
better make this a function which you call with argument. So you don't have to write multiple sfctl commands.